asteroidsyncserviced/main.cpp

```diff
@@ -22,10 +22,12 @@
 #include "dbusinterface.h"
 #include "watchesmanager.h"
 
-#ifdef UBUNTU_TOUCH_PLATFORM
-#include "platforms/ubuntutouch/ubuntuplatform.h"
+#if UBUNTU_TOUCH_PLATFORM
+#include "ubuntuplatform.h"
+using Platform = UbuntuPlatform;
 #elif SAILFISHOS_PLATFORM
-#include "platforms/sailfishos/sailfishplatform.h"
+#include "sailfishplatform.h"
+using Platform = SailfishPlatform;
```

Comment:
Ok.
But notice that this does not fit in the scope of what your commit describes. Best practices in version control systems is to have atomic commits https://tekin.co.uk/2021/01/how-atomic-commits-make-you-a-better-coder

Reply:
True, but the code won't compile without that change so I closed a previous PR to combine these.  Would you have preferred that I keep the other one (which was *only* this change) and make this PR dependent on that other one?